### ISSUE TITLE
Create a organisation documentation page

### ISSUE THREAD

**Issue by mkao006** [SWS-Methodology/Standards#19]
Probably should serve the documentations like in http://keras.io/.

and also serve them using the [organisation page](https://help.github.com/articles/user-organization-and-project-pages/)

@sebastian-c  what do you think? I know there is more or less of a page setup by @bowerth, but I would prefer not to add to the software stacks and also easy the documentation process.

**Comment by marcosmilzo** [comment]
Indeed pages like keras.io are of easy reading for a user guide.

**Comment by mkao006** [comment]
Also, since we will be writing in `.Rmd` or `.md` we can easily turn them into [Gitbook](http://jason.bryer.org/Rgitbook/) and have a properly structured book

**Comment by sebastian-c** [comment]
So you're suggesting that we convert our existing md files to a Gitbook? It's already [gitbook compatible](https://sebastian-c.gitbooks.io/r-sws-standards/content/index.html).

**Comment by mkao006** [comment]
Well, havent actually used it before but I think its more of restructuring currently available markdowns. We don't need to re- write them.

**Comment by mkao006** [comment]
Right, but more structured and have things like table of content and probably rename the repo as well.

**Comment by sebastian-c** [comment]
There is a table of contents. I think the repo needs to be restarted and restructured so as to more logically fit a new table of contents (or at least cleaned of the huge orange book)

**Comment by mkao006** [comment]
I would probably consider what kind of documentation we need before the structuring (which is long over due). 

The orange book is a methodological documentation, and I would probably have a separate book and re- write it to update the changes in the methodology.

While the organisation of the implementation including standards should be organised in a "developer's guide".

In addition, these documentation should be versioned inline with the implementation.
